tasks/release.py

```diff
@@ -558,7 +560,25 @@ def _get_windows_ddnpm_release_json_info(release_json, agent_major_version, is_f
 
     print(f"The windows ddnpm version is {win_ddnpm_version}")
 
-    return win_ddnpm_driver, win_ddnpm_version, win_ddnpm_shasum
+    # Step 2: DDPROCMON
```

Comment:
Not necessarily easier but a small method could have helped as it's mostly duplication of the above

Reply:
I guess that makes sense. Refactored.